src/webgpu/shader/execution/expression/binary/bitwise_shift.spec.ts

```diff
@@ -50,12 +45,13 @@ function is_valid_const_shift_left(e1: number, e1Type: string, e2: number) {
 }
 
 // Returns true if e1 >> e2 is valid for const evaluation
-function is_valid_const_shift_right(e1: number, e1Type: string, e2: number) {
+function isValidConstShiftRight(_e1: number, e1_type: ScalarType, e2: number) {
```

Comment:
General observation that this file uses a lot of `snake_case`, when the CTS should use `camelCase`.